extensions/import/src/test/wizard/pages/summaryPage.test.ts

```diff
@@ -0,0 +1,163 @@
+		should.notEqual(summaryPage.table, undefined);
+		should.notEqual(summaryPage.statusText, undefined);
+		should.notEqual(summaryPage.loading, undefined);
+		should.notEqual(summaryPage.form, undefined);
+
+		//Calling the clean up code
+		await summaryPage.onPageLeave();
+		await summaryPage.cleanup();
+
+	});
+
+	it('handle import updates status Text correctly', async function() {
+
+		// Creating a test Connection
+		let testServerConnection: azdata.connection.Connection = {
+			providerName: 'testProviderName',
+			connectionId: 'testConnectionId',
+			options: {}
+		};
+
+
+		// setting up connection objects in model
+		mockImportModel.object.server =  testServerConnection;
+		mockImportModel.object.database = 'testDatabase';
+		mockImportModel.object.schema = 'testSchema';
+		mockImportModel.object.filePath = 'testFilePath';
+
+		// Creating test columns
+		let testProseColumns = [
+			{
+				columnName: 'column1',
+				dataType: 'nvarchar(50)',
+				primaryKey: false,
+				nullable: false
+			},
+			{
+				columnName: 'column2',
+				dataType: 'nvarchar(50)',
+				primaryKey: false,
+				nullable: false
+			}
+		];
+		mockImportModel.object.proseColumns = testProseColumns;
+
+		// setting up a test table insert response from FlatFileProvider
+		let testSendInsertDataRequestResponse: InsertDataResponse = {
+			result: {
+				success: true,
+				errorMessage: ''
+			}
+		};
+		mockFlatFileProvider.setup(x => x.sendInsertDataRequest(TypeMoq.It.isAny())).returns(async () => { return testSendInsertDataRequestResponse; });
+
+		// Opening the wizard and initializing the page as SummaryPage
+		await new Promise(function (resolve) {
+			page.registerContent(async (view) => {
+				summaryPage = new SummaryPage(mockFlatFileWizard.object, page, mockImportModel.object, view, mockFlatFileProvider.object, mockApiWrapper.object);
+				pages.set(1, summaryPage);
+				await summaryPage.start().then(async () => {
+					summaryPage.setupNavigationValidator();
+
+					resolve();
+				});
+			});
+			wizard.generateScriptButton.hidden = true;
+
+			wizard.pages = [page];
+			wizard.open();
+		});
+
+		// Entering the page. This method will try to create table using FlatFileProvider
+		await summaryPage.onPageEnter();
+
+		// In case of success we should see the success message
+		should.equal(summaryPage.statusText.value,  constants.updateText);
+
+		// In case of a failure we should see the error message
+		testSendInsertDataRequestResponse = {
+			result: {
+				success: false,
+				errorMessage: 'testError'
+			}
+		};
+
+		// mocking the insertDataRequest to fail
+		mockFlatFileProvider.setup(x => x.sendInsertDataRequest(TypeMoq.It.isAny())).returns(async () => { return testSendInsertDataRequestResponse; });
+
+		// Entering the page. This method will try to create table using FlatFileProvider
+		await summaryPage.onPageEnter();
+		should.equal(summaryPage.statusText.value,  '✗ testError');
```

Comment:
Is this text available as a constant somewhere? Would be best not to have to update this too if that value ever changes